stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -167,6 +168,24 @@ public View getViewAndBoundsForHighlighting(View element, Rect bounds) {
     return element;
   }
 
+  @Nullable
+  @Override
+  public Object getElementToHighlightAtPosition(View element, int x, int y, Rect bounds) {
+    if (element instanceof ViewGroup) {
```

Comment:
Good question. No good reason as far as I can tell. Will update in the morning. Thanks!